src/components/PostMedia.tsx

```diff
@@ -13,39 +13,32 @@ interface PostMediaProps {
 
 const PostMedia = ({ attachment }: PostMediaProps): JSX.Element => {
   const getPostMediaItems = () => {
-    return attachment?.map(
-      (
-        item: {
-          type: string;
-          mediaType: string;
-          url: string | ActivityContentLink;
-        },
-        index: number
-      ) => (
+    return attachment?.map((item, index) => {
+      return (
         <div key={index} className="PostMedia__cover">
           {item.type.toLowerCase() === "image" && (
             <a
-              href={item.url as string}
+              href={item.url[0].href}
```

Comment:
I'm sure there needs to be some selection process here, but I think that's intended to be left as an exercise for the developer.

Reply:
yeah, ideally we should scan over each link and choose the one with...

1) the smallest resolution larger than size we intend to display, and
2) the highest quality format supported in the current browser

but I think we can save that for a future PR